aibolit/patterns/er_class/er_class.py

```diff
@@ -1,14 +1,15 @@
-import javalang
-
+from aibolit.ast_framework import ASTNodeType, AST
 from aibolit.utils.ast_builder import build_ast
+from typing import List
 
 
 class ErClass:
```

Comment:
Ununderstandable name. Better be changed. Also a comment describing a pattern in details may be added to this class.